lib/model/database.dart

```diff
@@ -186,6 +229,17 @@ class AppDatabase extends _$AppDatabase {
     return await (select(globalSettings)..limit(1)).getSingle();
   }
 
+  Future<Map<BoolGlobalSetting, bool>> getBoolGlobalSettings() async {
+    final result = <BoolGlobalSetting, bool>{};
+    final rows = await select(boolGlobalSettings).get();
+    for (final row in rows) {
+      final setting = BoolGlobalSetting.byName(row.name);
+      if (setting == null) continue;
```

Comment:
I think it will be beneficial to have tests verify that this doesn't throw when there are rows with unknown setting names.

Reply:
Hmm yeah, good catch. In this version no tests fail if I cut this line and say `setting!` on the next one.